source/index.blade.php

```diff
@@ -555,6 +559,11 @@ class="ud-contact-form-wrapper wow fadeInUp"
     <script src="assets/js/wow.min.js"></script>
     <script src="assets/js/main.js"></script>
     <script>
+      // ==== back to top
+      backToTop() {
+        window.scrollTo(0, 0);
+      }
```

Comment:
Looking at current site libresign.coop, have a nice effect implementing smooth.

I also suggest to make the JS separated from HTML using ID and adding the onclick event to element.

The code will be a bit bigger than the previous but will separate HTML and JS.

```suggestion
        document.getElementById('back-to-top').onclick = function() {
            e.preventDefault()
            window.scrollTo({
                top: 0,
                left: 0,
                behavior: 'smooth'
            })
        };
```